pyproject.toml

```diff
@@ -59,7 +61,9 @@ httpx = "^0.23.1"
 types-setuptools = "^67.2.0.1"
 textual-dev = "^1.1.0"
 pytest-asyncio = "*"
-pytest-textual-snapshot = "0.2.0"
+pytest-textual-snapshot = "0.2.0"  # TODO: pinned while while we resolve issue #3042
```

Comment:
Just to note: Once @willmcgugan deploys the latest (0.4.0) https://github.com/Textualize/pytest-textual-snapshot to PyPi, this should not be an issue.